mpv

```diff
@@ -106,12 +106,12 @@ start_mpv(){
 	done
 	[ -z "${1}" ] && err 'Exception: No Video URL provided'
 	printf '%s' "${1}" | grep -qEo "^(file://|http://|https://|ytdl://|smb://|bd://)[^'\"<>]*|^-$" || err "Invalid type of URL"
-	if printf '%s' "${1}" | grep -qEo "file://[^'\"<>]*" && [ -e "${1#file://}" ]; then
-		set -- "file://$(realpath "${1#file://}")"
-	else
-		err "mpv: ${1#file://}: No such file or directory"
+	if printf '%s' "${1}" | grep -qEo "file://[^'\"<>]*"; then
+		path_file="$(realpath "${1#file://}")"
+		[ -e "${path_file#file://}" ] && err "mpv: ${path_file#file://}: No such file or directory"
```

Comment:
Sorry for being dumb lol, this is why you're getting `No such file directory` error. I put `&&` instead of `||`, the latest should work for you now.

Reply:
Also omit `-nc`, needless to use netcat when working with `phone storage/external storage`